activerecord/lib/active_record/enum.rb

```diff
@@ -119,8 +132,9 @@ def serialize(value)
       attr_reader :name, :mapping
     end
 
-    def enum(definitions)
+    def enum(definitions, options = {})
```

Comment:
Double hash is definitely a no-go. I'm OK with prefix/suffix being reserved keywords for this method under the circumstances.